ff4j-web/src/main/java/org/ff4j/web/controller/PropertiesController.java

```diff
@@ -128,9 +128,10 @@ public void post(HttpServletRequest req, HttpServletResponse res, WebContext ctx
             } else {
                 Property<?> p = getFf4j().getPropertiesStore().readProperty(propertyName);
                 Property<?> newProperty = PropertyFactory.createProperty(newName, p.getType(), p.asString(), p.getDescription(), null);
-                for(Object o : p.getFixedValues()) {
-                    newProperty.add2FixedValueFromString(o.toString());
```

Comment:
Here, the o.toString() call, in case of Date objects, is doing the marshaling with Date.toString(). This should be done with PropertyDate.asString() such that it is matching the later call to unmarshaling done with PropertyDate.fromString(). Because of different format for Date.toString() compared with PropertyDate.fromString() an error related with wrong format is raised on unmarshaling.